README.md

```diff
@@ -122,9 +122,9 @@ All commands use discords slash commands (/)
 
 ```/vote_draft``` 
 
-Upon typing the command the bot will present you with three buttons: Create, Edit, Delete.
+Upon typing the command the bot will present you with three buttons: Create, Edit, Preview, and Delete. 
 
-These should be self explained, create will allow you to draft a proposal. Edit will allow you to edit an existing proposal, and delete
+These should be self explained, clicking create will prompt you with either general or budget, these options dictate the type of proposal. Edit will allow you to edit an existing proposal, and delete
```

Comment:
```
"Create" will create a new proposal. Note: "Proposal Type" field should be either "General" or "Budget".
"Edit" will allow you to edit an existing proposal.
"Preview" will allow you to preview an existing proposal before it is published.
"Delete" will let you delete and existing proposal that has not been published.
```

Reply:
Sounds good

> I think this would be better written like this (no need to explain the "general" "budget" selection):
> 
> ```
> "Create" will create a new proposal.
> "Edit" will allow you to select an an existing proposal to edit.
> "Preview" will allow you to preview an existing proposal before it is published.
> "Delete" will allow you delete and existing proposal that has not been published.
> ```